### ISSUE TITLE
Question: Variation Reporting (using attributes and terms)

### ISSUE THREAD

**Issue by myfriendmike** [woocommerce/wc-admin#1562]
Is the idea that the advanced reporting will include the option to run a report on product variations based on the terms and attributes used to create the variable product in the first place?..or even by a specific ID (or range of IDs) of a variation?

Just installed v 0.7.0 and Product: Includes using Advance Filters only seems to use the title of the parent (is that the correct term to use) product thus far.

I can't imagine that we are alone on this one but we have a product that has 96 variations. Trying to run reports on which of these actually sold and how many doesn't seem possible without plugins currently. Would be great if this was the long term plan with WC-Admin. Any info on this would be great to hear.

**Comment by psealock** [comment]
Hi @myfriendmike, thanks for your question!

Yes, we're reporting on product variations based on variations created from attributes.

I just had a look and it appears there is a regression with variation labels. There should be labels based off of the parent and variation attributes like "Jeans / boot cut, stone wash".

### Steps to reproduce

1. Create a variation of a product based on 2 attributes (maybe this is a problem with just 1?)
2. Create an order
3. Products Report > Show > Single Product > choose product
4. See the chart and table render with only the parent's name.
5. See the labels include only the parent product name
6. See Summary Numbers and Table values not add up

![screen shot 2019-02-15 at 2 54 31 pm](https://user-images.githubusercontent.com/1922453/52829506-f845e700-3131-11e9-8ec7-f0dbd9922529.png)

**Comment by myfriendmike** [comment]
Thanks @psealock 

It's possible i wasn't perfectly clear with the question...or that I don't understand what the regression means!

My question was really whether it would be possible to run a report that would return all (sales of ) variations of a product based on JUST ONE, or a combination of 2 or more of the attributes or more importantly terms.

So for example - theres a framed photo product and variations based on the following attributes:

Date of event (for photo to mount in frame) - 4 options/terms ( 1st, 2nd, 3rd, 4th).
Number of apertures available in the mount inserted into the frame - 2 options/terms (2 or 3 apertures).
Colour of frame - 3 options/terms (Red, White, Blue).

(From using woocommerce so far I understand that the name of the product doesn't change for the variation, just the attributes it is made up of.)

Will it be possible to report that returns ALL sales of this single product based on whether the Date was the 1st.

Will it be possible to report that returns ALL sales of this single product based on whether the Date was the 1st and the mount had 2 apertures

Will it be possible to report that returns ALL sales of this single product based on whether the Date was the 1st, whether the mount had 2 apertures and the colour of the frame was Red.

Hope that makes sense as a question.

**Comment by justinshreve** [comment]
@psealock I don't seem to see the same issue as you. How was the `Belt` orders place? Manually, or via Smooth Generator? I had issues with smooth generator variation orders not generating correctly, however if you place the order manually, they show up correctly.

<img width="1323" alt="screen shot 2019-02-15 at 8 45 50 am" src="https://user-images.githubusercontent.com/689165/52860738-4b9e4200-30fe-11e9-8a26-c60b0712437d.png">
<img width="1331" alt="screen shot 2019-02-15 at 8 45 01 am" src="https://user-images.githubusercontent.com/689165/52860740-4b9e4200-30fe-11e9-90db-48418e326908.png">

---

@myfriendmike 

> My question was really whether it would be possible to run a report that would return all (sales of ) variations of a product based on JUST ONE, or a combination of 2 or more of the attributes or more importantly terms.

This is an interesting suggestion, but not one currently on our road-map. You can currently only pull up variations based on name and not based on attributes. cc @LevinMedia for consideration in the future. The Gutenberg Product Blocks allow you to search based on an attribute, so that could be worth bringing over to the advanced filters as well.

**Comment by psealock** [comment]
@myfriendmike 

Thanks for the clarification! The answer above is correct and your question has noted a feature we'd like to evaluate for potential future development. I am closing this issue for now.

@justinshreve 

>How was the `Belt` orders placed?

I created the variations and placed the orders manually. I tried again and can't reproduce, its working as expected. I previously had other variations of `Belt` that I removed before creating new ones, so this may be a separate issue.
